### ISSUE TITLE
Is restriction from README about Java8 up-to-date?

### ISSUE THREAD

**Issue by php-coder** [jcabi/jcabi-http#148]
At the end of `README` file I read the following:
```
Make sure you're using Maven 3.2+ and Java7 (in Java8 you won't be able to use Qulice, because of teamed/qulice#379).
```
As I can see teamed/qulice#379 is fixed, so, most likely, this isn't an issue anymore.

**Comment by dmarkov** [comment]
@yegor256 dispatch this issue please, see [par.21](http://at.teamed.io/policy.html#21)

**Comment by yegor256** [comment]
@php-coder fixed in 01f6498 thanks!
